src/site/content/en/blog/referrer-best-practices/index.md

```diff
@@ -304,7 +303,7 @@ fetch(url, {referrerPolicy: 'no-referrer-when-downgrade'});
 ```
 
 {% Aside 'gotchas' %} A per-element policy is [not supported in all
-browsers](https://caniuse.com/#search=referrerpolicy). But browsers that don't support this tend to
+browsers](https://caniuse.com/#feat=mdn-api_request_referrerpolicy). But browsers that don't support this tend to
```

Comment:
@kaycebasques API Request Policy is different from per-element policy (HTML). But I see why you'd like a more precise link ==> Can we replace this with:

A per-element policy is not supported in all
browsers (Example: [referrerpolicy on an <a> element](https://caniuse.com/#feat=mdn-html_elements_a_referrerpolicy), [on an `img` element](https://caniuse.com/#feat=mdn-html_elements_img_referrerpolicy) and [on an `link` element](https://caniuse.com/#feat=mdn-html_elements_link_referrerpolicy))